app/assets/javascripts/pageflow/editor/views/edit_meta_data_view.js

```diff
@@ -34,9 +34,16 @@ pageflow.EditMetaDataView = Backbone.Marionette.Layout.extend({
       });
 
       this.input('credits', pageflow.TextAreaInputView);
-      this.input('author', pageflow.TextInputView);
-      this.input('publisher', pageflow.TextInputView);
-      this.input('keywords', pageflow.TextInputView);
+
+      this.input('author', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_author
+      });
+      this.input('publisher', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_publisher
+      });
+      this.input('keywords', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_keywords
+      });
```

Comment:
Looking at [the meta tag helper](https://github.com/scrollytelling/pageflow/blob/8bec49b35e67fdb47a0326834e49fa46cee4d798/app/helpers/pageflow/meta_tags_helper.rb) it looks like we are actually falling back to global config, not the theming default when the user clears an input here. So I guess, we should be should be displaying that as a placeholder.